roles/confluent.variables/defaults/main.yml

```diff
@@ -7,6 +7,12 @@ confluent_full_package_version: "{{ confluent_package_version + '-1' }}"
 confluent_package_redhat_suffix: "{{ '-' + confluent_full_package_version if confluent_full_package_version != '' else ''}}"
 confluent_package_debian_suffix: "{{ '=' + confluent_full_package_version if confluent_full_package_version != '' else ''}}"
 
+### Boolean to mask secrets in playbook output
+mask_secrets: true
+
+### Boolean to mask output generated by diff flag
+mask_diff: true
```

Comment:
Lets rename it to be what it is, something like `mask_sensitive_diff` 
mask_diff as a variable name makes it seem like well mask all diff

Reply:
Done!

Reply:
could also use this as a good moment to improve the other variable name... mask_secrets is pretty confusing bc we have a feature called secrets protection. I would do something like 

```
# Deprecated, see mask_sensitive_logs
mask_secrets: true

### Boolean to mask secrets in playbook output
mask_sensitive_logs: "{{mask_secrets}}"

### Boolean to mask output generated by diff flag
mask_sensitive_diff: true
```

then do a find and replace in all other files mask_secrets -> mask_sensitive_logs

This way both variables have the same naming convention.
Notice that you have to keep the original variable around for backwards compatibility

Reply:
Good idea. Done the changes.

Reply:
would  it not be better this way - 
```
# Deprecated, see mask_sensitive_logs
mask_secrets: "{{mask_sensitive_logs}}"

### Boolean to mask secrets in playbook output, defaults to true
mask_sensitive_logs: true
```